site/Makefile

```diff
@@ -0,0 +1,79 @@
+# Credit to the Kubernetes Website team. (https://github.com/kubernetes/website/blob/main/Makefile)
+HUGO_VERSION      = $(shell grep ^HUGO_VERSION netlify.toml | tail -n 1 | cut -d '=' -f 2 | tr -d " \"\n")
+NODE_BIN          = node_modules/.bin
+NETLIFY_FUNC      = $(NODE_BIN)/netlify-lambda
+
```

Comment:
```suggestion

```
I think those were leftovers from the k/website Makefile. I didn't find usages other than the declarations.

Reply:
Removed.